code/espurna/sensors/VEML6075Sensor.h

```diff
@@ -0,0 +1,96 @@
+// -----------------------------------------------------------------------------
+// VEML6075 Sensor over I2C
+// Copyright (C) 2017-2018 by Xose Pérez <xose dot perez at gmail dot com>
+// -----------------------------------------------------------------------------
+
+#if SENSOR_SUPPORT && VEML6075_SUPPORT
+
+#pragma once
+
+#undef I2C_SUPPORT
+#define I2C_SUPPORT 1 // Explicitly request I2C support.
+
+
+#include "Arduino.h"
+#include "I2CSensor.h"
+#include "SparkFun_VEML6075_Arduino_Library.h"
+
+class VEML6075Sensor : public I2CSensor {
+
+    public:
+
+        // ---------------------------------------------------------------------
+        // Public
+        // ---------------------------------------------------------------------
+
+        VEML6075Sensor(): I2CSensor() {
+            _count = 3;
+            _sensor_id = SENSOR_VEML6075_ID;
+            _veml6075 = new VEML6075();
+        }
+
+        ~VEML6075Sensor() {
+          delete _veml6075;
```

Comment:
@xoseperez I don't know if this is enough to have on the destructor..

Reply:
Should be. I have not dive into the VEML6075 library code.